app/src/features/Batches/BatchListItem.tsx

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+
+import { Link, Paper, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { Replay } from "@material-ui/icons";
+import moment from "moment";
+import { useTranslation } from "react-i18next";
+
+import Button from "common/components/Button";
+import { Batch } from "services/api/generated/api.generated";
+
+import { KIBANA_URL } from "../../constants";
+import BatchCancel from "./BatchCancel";
+
+type BatchListItemType = {
+  batch: Batch;
+};
+
+const useStyles = makeStyles((theme) => ({
+  listItem: {
+    marginBottom: theme.spacing(0.5),
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    padding: theme.spacing(1.3, 2, 1, 2),
+    flexWrap: "wrap",
+  },
+  listItemActions: {
+    display: "flex",
+    alignItems: "center",
+    flexWrap: "wrap",
+  },
+  margin: {
+    marginRight: theme.spacing(1),
+  },
+  title: {
+    display: "flex",
+  },
+}));
+
+const createKibanaLink = (batchCreation: string, batchEnd: string) => {
+  return `${KIBANA_URL}/app/kibana#/discover?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'${batchCreation}',to:'${batchEnd}'))&_a=(columns:!(_source),filters:!(),interval:auto,query:(language:kuery,query:error),sort:!())`;
+};
+
+const BatchListItem = ({ batch }: BatchListItemType): JSX.Element => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const batchEnd = batch.completed_at ?? batch.canceled_at;
+
+  const getBatchDuration = (batch: Batch) => {
+    if (batchEnd) {
+      const start = moment(batch.created_at);
+      const end = moment(batchEnd);
+      const diff = moment.duration(end.diff(start));
+      return `${diff.years() ? `${t("year", { count: diff.years() })} ` : ""}${
+        diff.months() ? `${t("month", { count: diff.months() })} ` : ""
+      }${diff.days() ? `${t("day", { count: diff.days() })} ` : ""}${
+        diff.hours() ? `${t("hour", { count: diff.hours() })} ` : ""
+      }${diff.minutes() ? `${t("minute", { count: diff.minutes() })} ` : ""}${
+        diff.seconds() ? `${t("second", { count: diff.seconds() })} ` : ""
+      }`;
+    }
+  };
+
+  const getCreatedAtDate = (date: string) =>
+    new Date(date).toLocaleString().split(",").join(" -");
+
+  const handleBatchRetry = (batchId: string) => (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+  ) => {
+    e.stopPropagation();
+    // TODO: batch retry is not implemented yet
+    console.log(batchId);
+  };
+
+  return (
+    <Paper
+      key={batch.id}
+      className={classes.listItem}
+      variant="outlined"
+      elevation={2}
+    >
+      <div>
+        <div className={classes.title}>
+          <Typography variant="subtitle2">
+            {!batch.completed_at && !batch.canceled_at && t("batchInProgress")}
+            {batch.completed_at && batch.errors.length > 0 && (
```

Comment:
I'd create 2 booleans to specify those conditions in a more verbose but understandable way : 
```suggestion
            {isBatchInProgress && t("batchInProgress")}
            {hasBatchCompletedWithErrors && (
```